src/k_lsm/k_lsm_inl.h

```diff
@@ -82,6 +82,55 @@ k_lsm<K, V, Rlx>::insert(const K &key,
 
 template <class K, class V, int Rlx>
 bool
+k_lsm<K, V, Rlx>::delete_min(K &key, V &val)
```

Comment:
Please don't duplicate all of this code. Why not implement delete_min(val) as 

delete_min(V &val) {
  K key;
  return delete_min(key, val);
}

Same in all other files of this commit.